reactions/src/SPT.cpp

```diff
@@ -843,6 +910,90 @@ for(int zi = 0; zi<noz; zi++){
 }
 }
 
+// BILL MOD
```

Comment:
This function needs reassessment - P(k,z)  is being passed by cosmosis in a 2D array , while the class assumes a 1d array is being fed. We can think about this later - should probably comment or leave out this function for the time being since it is not computing things correctly.

Reply:
Ok, I've commented the function out for now and we can address it properly in a future update.